.gitignore

```diff
@@ -3,3 +3,5 @@
 !*.env.example
 node_modules
 dist
+*.bat
+*.sh
```

Comment:
Warum ignorierst du die? Falls es irgendwelche Skripte sind, die du nur lokal benutzt, dann mach doch lieber einen ".local" Ordner auf Root-Ebene, in den du die rein packst.

Reply:
Das sind primär lokale Skripte um das TUI für verschiedene Connectoren mittels Environment Variables zu steuern. env.example ist auch in der Gitignore, warum dann nicht diese Skripte? ;)

Reply:
.env.example wird nicht ignored. Im Gegenteil, es wird geunignored, weil ja .env.* ignored wird. 
Und env-Files sind halt was allgemeingültiges, und es kann riiichtig böse enden, wenn die gepushed werden.

*.sh/*.bat zu ignoren finde ich nicht ganz so geschickt, weil sich in Zukunft dann vielleicht jemand wundert, warum sein Script-File, was er eigentlich committen möchte, nicht in der Liste geänderter Dateien auftaucht. Falls er es überhaupt merkt.